torch/_inductor/ir.py

```diff
@@ -3830,13 +3830,26 @@ def codegen(self, wrapper):
             kernel, configs, self.kwargs
         )
 
+        args = self.codegen_kwargs()
+        if V.graph.cpp_wrapper:
+            from triton.runtime.autotuner import Autotuner
+            from triton.runtime.jit import JITFunction
+
+            if isinstance(kernel, Autotuner):
+                kernel = kernel.fn
```

Comment:
I'm still a bit confused about your comment there.
```
kernel, configs = self.get_kernel_and_configs()
```
is guaranteed to return a kernel such that
```
assert isinstance(kernel, JITFunction)
```
if you look at line 3820 above, it does kernel = kernel.fn when it is autotuner.

Reply:
Yeah, I think, you're right. The autotuning test has failed before when I was fetching the `kernel` myself. Now that I'm reusing the `kernel` returned by `get_kernel_and_configs`, I can skip the unwrapping as it's already done.